WPMedia/fusion-news-theme-blocks
# 2.1.2 -> 2.1.3 Merge changes

## Description

This PR merges changes from 2.1.2 into 2.1.3

## Jira Ticket

- [THEMES-1600](https://arcpublishing.atlassian.net/browse/THEMES-1600)

## Test Steps

1. Checkout this branch `git checkout THEEMS-1600`
2. Run fusion repo with linked blocks `npx fusion start -f -l`
3. Sites should build okay, and check the all blocks page in a couple of different languages to make sure the merge conflicts from all of the `intl.json` files resolved correctly.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1600]: https://arcpublishing.atlassian.net/browse/THEMES-1600?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
